### PR TITLE
More haml clean up

### PR DESCRIPTION
**Pull request by Br3nda**

### CODE REVIEW

**Comment by Br3nda** [comment]
Many of these commits are already in the destination -- github is tripping because the were rebased

**Comment by Br3nda** [comment]
sorted. Github showing sensible list of commits now

File: app/views/seeds/index.rss.haml [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-      Recent seeds from #{ @owner ? @owner : 'all members' } (#{ENV['GROWSTUFF_SITE_NAME']})
+      Recent seeds from #{@owner ? @owner : 'all members'} (#{ENV['GROWSTUFF_SITE_NAME']})
     %link= seeds_url
     - @seeds.each do |seed|
       %item
         %title #{seed.owner}'s #{seed.crop} seeds
-        %pubDate= seed.created_at.to_s(:rfc822)
+        %pubdate= seed.created_at.to_s(:rfc822)
         %description
           :escaped
```
Comment (Br3nda) [inline app/views/seeds/index.rss.haml]: I couldn't see why this is escaped.... Maybe it can be removed.

File: app/views/seeds/index.rss.haml [hunk 2]
```diff
@@ -16,8 +16,9 @@
             <p>GMO? #{seed.gmo}</p>
             <p>Heirloom? #{seed.heirloom}</p>
           - if seed.tradable?
-            :escaped
-              <p>Will trade #{seed.tradable_to} from #{seed.owner.location ? seed.owner.location : 'unknown location'}</p>
+            %p
```
Comment (Br3nda) [inline app/views/seeds/index.rss.haml]: I changed this to be escaped properly. I'm unsure how clean our location fields are.

Comment (pozorvlak) [inline app/views/seeds/index.rss.haml]: Where's the escaping happening now?

Our location fields appear to be [completely unclean](http://www.staging.growstuff.org/members/pozorvlak). Oops.

```
irb(main):002:0> me = Member.find('pozorvlak')
  Member Load (2.3ms)  SELECT  "members".* FROM "members" WHERE "members"."slug" = $1  ORDER BY lower(login_name) asc LIMIT 1  [["slug", "pozorvlak"]]
  Member Load (2.3ms)  SELECT  "members".* FROM "members" WHERE "members"."slug" = $1  ORDER BY lower(login_name) asc LIMIT 1  [["slug", "pozorvlak"]]
irb(main):003:0> me.location
=> "<script>alert('Hello, world!')</script>"
```

Comment (Br3nda) [inline app/views/seeds/index.rss.haml]: the haml will escape it. (as long as you don't put `:escaped` in front of it like we did)

Comment (Br3nda) [inline app/views/seeds/index.rss.haml]: Let's remove all `:escaped` from our code base. I'll do this today

side note: our spammer is likely watching these conversations.

**Comment by Br3nda** [comment]
@pozorvlak are you happy? I'd like to get this merged and then send a haml-escaping-clean up

**Pull request merged**